primitives/src/helpers.rs

```diff
@@ -195,7 +188,9 @@ pub fn read_g1_point_from_bytes_be(g1_bytes_be: &[u8]) -> Result<G1Affine, &str>
 
     if m_data == m_compressed_infinity {
         if !is_zeroed(g1_bytes_be[0] & !m_mask, g1_bytes_be[1..32].to_vec()) {
-            return Err("point at infinity not coded properly for g1");
+            return Err(KzgError::SerializationError(
+                "point at infinity not coded properly for g1".to_string(),
```

Comment:
can we create an incorrectSRSPoints error or DeserializatonError or something for these? SInce they are very different from "I failed to serialize the data" errors